### PR TITLE
pkp/pkp-lib#5504 Control access to settings by user group

### PR DESCRIPTION
**Pull request by asmecher**

### CODE REVIEW

**Review by bozana — COMMENTED** [review]
Hi @asmecher, the same/similar comment as in PKP-LIB PR...
Thanks a lot!

File: registry/userGroups.xml [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 <groups>
 	<!-- Managerial Role -->
-	<group roleId="0x00000010" name="default.groups.name.manager" plural="default.groups.plural.manager" abbrev="default.groups.abbrev.manager" permitMetadataEdit="true" />
-	<group roleId="0x00000010" stages="1,3,4,5" name="default.groups.name.editor" plural="default.groups.plural.editor" abbrev="default.groups.abbrev.editor" permitMetadataEdit="true" masthead="true" />
+	<group roleId="0x00000010" name="default.groups.name.manager" plural="default.groups.plural.manager" abbrev="default.groups.abbrev.manager" permitMetadataEdit="true" permitSettings="true" />
+	<group roleId="0x00000010" stages="1,3,4,5" name="default.groups.name.editor" plural="default.groups.plural.editor" abbrev="default.groups.abbrev.editor" permitMetadataEdit="true" masthead="true" permitSettings="true" />
 	<group roleId="0x00000010" stages="4,5" name="default.groups.name.productionEditor" plural="default.groups.plural.productionEditor" abbrev="default.groups.abbrev.productionEditor" permitMetadataEdit="true" />
```
[the remark this answers is not in the document]
Comment (bozana) [reply registry/userGroups.xml:19]: I see in OMP this attribute is set to true for 'Production editor', thus, I suppose it should be the same here?

[the remark this answers is not in the document]
Comment (asmecher) [reply registry/userGroups.xml]: Thanks -- yes, I fixed it in OMP but missed it here. Resolved!

**Comment by asmecher** [comment]
> EDIT: are we supposed to commit pkp.min.js? 🤔

Yes, we rarely modify those scripts these days, but it's a good habit when we do. (The build process will catch us if we forget, though.)

**Pull request closed without merging**